components/dashboard/src/settings/Plans.tsx

```diff
@@ -433,7 +433,7 @@ export default function () {
                 {!assignedTs && (
                     <p className="text-base w-96 m-auto">Upgrade your plan to get access to private repositories or more parallel workspaces.</p>
                 )}
-                <p className="mt-2 font-semibold text-gray-500">Remaining hours: {typeof(accountStatement?.remainingHours) === 'number'
+                <p className="mt-2 font-semibold text-gray-500" title={guessCurrentBillingCycle(currentPlan, accountStatement).map(d => d.toLocaleDateString()).join(' - ')}>Remaining hours: {typeof(accountStatement?.remainingHours) === 'number'
```

Comment:
**thought**: It took me a while to realize this was added as a title attribute! 😇

Reply:
Titles are fantastic! I always hover over UI elements when I need more info, and pretty often I'm not disappointed. 😁

If we already have a tooltip component, I think we could do some JS-hacking to make it work for every element that has a `title` attribute.

Reply:
> Titles are fantastic!

Agree, and `title` along with `aria-label` attributes can significantly improve product accessibility. 👀 

> If we already have a tooltip component ...

We do have first [tooltip component](https://github.com/gitpod-io/gitpod/blob/4850f7c06fe45e6398b1fc53d0119d08839c8fc6/components/dashboard/src/components/Tooltip.tsx) implementation! 💬 

> I think we could do some JS-hacking ...

I'd not expect to see a tooltip element every time we're using the title attribute as this could make the user interface unnecessarily verbose, however, we can try this and see if we need to tone this down. 🏀